pkg/model/tangle/bundle_storage.go

```diff
@@ -101,7 +107,7 @@ func (bundle *Bundle) ObjectStorageValue() (data []byte) {
 	return value
 }
 
-func (bundle *Bundle) UnmarshalObjectStorageValue(data []byte) (consumedBytes int, err error) {
+func (bundle *Bundle) UnmarshalObjectStorageValue(data []byte) (err error) {
```

Comment:
why keep this?

Reply:
Less changes in the code before we refactor everything.